cypress/support/e2e.js

```diff
@@ -0,0 +1,5 @@
+import 'cypress-axe'
```

Comment:
Por convenção, separa-se _libs_ externas de módulos internos com uma linha em branco.

```suggestion
import 'cypress-axe'

```

Dessa forma, fica fácil diferenciá-las.